code/modules/surgery/organs/brain.dm

```diff
@@ -125,6 +125,65 @@
 	if(owner && vital)
 		owner.setBrainLoss(120)
 
+/obj/item/organ/internal/brain/on_life()
+	if(decoy_brain || damage < 10) 
+		return
+	switch(damage)
+		if(10 to 30)
+			handle_minor_brain_damage()
+		if(31 to 60)
+			handle_moderate_brain_damage()
+		if(61 to 80)
+			handle_severe_brain_damage()
+		if(81 to 100)
+			handle_critical_brain_damage()
+
+/obj/item/organ/internal/brain/proc/handle_minor_brain_damage()
+	if(prob(5))
+		owner.Dizzy(5 SECONDS)
+		to_chat(owner, "<span class='warning'>Your head feels foggy.</span>")
+	else if(prob(4))
+		owner.vomit()
+		to_chat(owner, "<span class='warning>'You feel nauseous.</span>")
+
+/obj/item/organ/internal/brain/proc/handle_moderate_brain_damage()
+	if(prob(4))
+		owner.Confused(20 SECONDS)
+		to_chat(owner, "<span class='warning'>It's suddenly difficult to walk straight.</span>")
+	else if(prob(5))
+		owner.EyeBlurry(15 SECONDS)
+		to_chat(owner, "<span class='warning'>Your vision unfocuses.")
+	else if(prob(3))
+		owner.Drowsy(20 SECONDS)
+		to_chat(owner, "<span class='warning'>You're getting tired.</span>")
+
+/obj/item/organ/internal/brain/proc/handle_severe_brain_damage()
+	if(prob(5))
+		owner.Hallucinate(60 SECONDS)
+		to_chat(owner, "<span class='warning'>You start losing your grip on reality.</span>")
+	else if(prob(10))
+		owner.Slur(60 SECONDS)
+		owner.Stuttering(60 SECONDS)
+		to_chat(owner, "<span class='warning'>You can't form your words properly.</span>")
+	else if(prob(2))
+		owner.Stun(5 SECONDS)
+		to_chat(owner, "<span class='warning'>You stare forward in a stupor.</span>")
+	else if(prob(5))
+		owner.KnockDown(1 SECONDS)
+		to_chat(owner, "<span class='warning'>You lose your footing, and stumble.</span>")
+
+/obj/item/organ/internal/brain/proc/handle_critical_brain_damage()
+	if(prob(4))
+		owner.Silence(45 SECONDS)
+		to_chat(owner, "<span class='warning'>You open your mouth to speak, but no sound comes out.</span>")
+	else if(prob(5))
+		owner.EyeBlind(30 SECONDS)
+		to_chat(owner, "<span class='warning'>Your vision gives out.</span>")
+	else if(prob(5))
+		owner.Weaken(10 SECONDS)
+		owner.Jitter(10 SECONDS)
```

Comment:
I think you might want to increase the jitter here like how epilepsy does it; it's based on time. You can probably set it back to zero on a timer.

Reply:
Epilepsy intends on being essentially a permanent jitter, that's why it's number is so high. Though it does feel somewhat fitting for critical brain damage to make you permanently jittery, as well.

Reply:
Yeah, what I mean is that jitter's intensity is based on the remaining time of the effect. It's silly, but it's how it's currently done. If you want to make them "seize", you're best off setting the jitter stupid high, and then resetting it on a timer.